fix(components): 针对于修改description中的rowspan的，导致跨行计算错误引起的填充失效

gyc-up opened the pull request:
之前的description代码在计算rowspan的跨行时，对于在上一行设置rowspan无法正确计算，会导致下一行并没有被正确填充，使用了rowspanMap重新对剩余行数和列数进行了计算，在样式实现上和之前无误

BREAKING CHANGE :
修改了之前description组件内的跨行计算规则

closed #18956

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

gyc-up committed:
fix(components): 针对于修改description中的rowspan的，导致跨行计算错误引起的填充失效

之前的description代码在计算rowspan的跨行时，对于在上一行设置rowspan无法正确计算，会导致下一行并没有被正确填充，使用了rowspanMap重新对剩余行数和列数进行了计算，在样式实现上和之前无误

BREAKING CHANGE :
修改了之前description组件内的跨行计算规则

closed #18956

tolking commented:
似乎还是存在一些问题

[Demo](https://element-plus.run/#eyJBcHAudnVlIjoiXG5cbjx0ZW1wbGF0ZT5cbiAgPGVsLWNhcmQgc3R5bGU9XCJtYXgtd2lkdGg6IDEwMCVcIj5cbiAgICA8ZWwtZGVzY3JpcHRpb25zIHRpdGxlPVwiQ3VzdG9taXplZCBzdHlsZSBsaXN0XCIgOmNvbHVtbj1cIjNcIiBib3JkZXI+XG4gICAgICA8ZWwtZGVzY3JpcHRpb25zLWl0ZW1cbiAgICAgICAgICBsYWJlbD1cInh4eHh4eFwiXG4gICAgICAgICAgbGFiZWwtYWxpZ249XCJyaWdodFwiXG4gICAgICAgICAgYWxpZ249XCJjZW50ZXJcIlxuICAgICAgICAgIGxhYmVsLWNsYXNzLW5hbWU9XCJteS1sYWJlbFwiXG4gICAgICAgICAgY2xhc3MtbmFtZT1cIm15LWNvbnRlbnRcIlxuICAgICAgICAgIHdpZHRoPVwiMTUwcHhcIlxuICAgICAgPlxuICAgICAgICBrb29yaW9va2FtaVxuICAgICAgPC9lbC1kZXNjcmlwdGlvbnMtaXRlbT5cbiAgICAgIDxlbC1kZXNjcmlwdGlvbnMtaXRlbSBsYWJlbD1cInh4eHhcIiBsYWJlbC1hbGlnbj1cInJpZ2h0XCIgYWxpZ249XCJjZW50ZXJcIj5cbiAgICAgICAgMTgxMDAwMDAwMDBcbiAgICAgIDwvZWwtZGVzY3JpcHRpb25zLWl0ZW0+XG4gICAgICA8ZWwtZGVzY3JpcHRpb25zLWl0ZW0gbGFiZWw9XCJ4eHhcIiBsYWJlbC1hbGlnbj1cInJpZ2h0XCIgYWxpZ249XCJjZW50ZXJcIj5cbiAgICAgICAgU3V6aG91XG4gICAgICA8L2VsLWRlc2NyaXB0aW9ucy1pdGVtPlxuICAgICAgPGVsLWRlc2NyaXB0aW9ucy1pdGVtIGxhYmVsPVwieHh4eFwiIGxhYmVsLWFsaWduPVwicmlnaHRcIiBhbGlnbj1cInJpZ2h0XCI+XG4gICAgICAgIDxlbC10YWcgc2l6ZT1cInNtYWxsXCI+U2Nob29sPC9lbC10YWc+XG4gICAgICA8L2VsLWRlc2NyaXB0aW9ucy1pdGVtPlxuICAgICAgPGVsLWRlc2NyaXB0aW9ucy1pdGVtIGxhYmVsPVwieHh4eHh4XCIgbGFiZWwtYWxpZ249XCJyaWdodFwiIGFsaWduPVwiY2VudGVyXCIgc3Bhbj1cIjJcIiByb3dzcGFuPVwiMlwiPlxuICAgICAgICBOby4xMTg4LCBXdXpob25nIEF2ZW51ZSwgV3V6aG9uZyBEaXN0cmljdCwgU3V6aG91LCBKaWFuZ3N1IFByb3ZpbmNlXG4gICAgICA8L2VsLWRlc2NyaXB0aW9ucy1pdGVtPlxuICAgICAgPGVsLWRlc2NyaXB0aW9ucy1pdGVtIGxhYmVsPVwieHh4eHh4eHhcIiBsYWJlbC1hbGlnbj1cInJpZ2h0XCIgYWxpZ249XCJjZW50ZXJcIj5cbiAgICAgICAgPGVsLXRhZyBzaXplPVwic21hbGxcIj5TY2hvb2w8L2VsLXRhZz5cbiAgICAgIDwvZWwtZGVzY3JpcHRpb25zLWl0ZW0+XG4gICAgICA8ZWwtZGVzY3JpcHRpb25zLWl0ZW0gbGFiZWw9XCJ4eHh4eHhcIiBsYWJlbC1hbGlnbj1cInJpZ2h0XCIgYWxpZ249XCJjZW50ZXJcIj5cbiAgICAgICAgTm8uMTE4OCwgV3V6aG9uZyBBdmVudWUsIFd1emhvbmcgRGlzdHJpY3QsIFN1emhvdSwgSmlhbmdzdSBQcm92aW5jZVxuICAgICAgPC9lbC1kZXNjcmlwdGlvbnMtaXRlbT5cbiAgICA8L2VsLWRlc2NyaXB0aW9ucz5cbiAgPC9lbC1jYXJkPlxuXG5cblxuPC90ZW1wbGF0ZT5cblxuPHN0eWxlIHNjb3BlZD5cbjpkZWVwKC5teS1sYWJlbCkge1xuICBiYWNrZ3JvdW5kOiB2YXIoLS1lbC1jb2xvci1zdWNjZXNzLWxpZ2h0LTkpICFpbXBvcnRhbnQ7XG59XG46ZGVlcCgubXktY29udGVudCkge1xuICBiYWNrZ3JvdW5kOiB2YXIoLS1lbC1jb2xvci1kYW5nZXItbGlnaHQtOSk7XG59XG48L3N0eWxlPlxuPHNjcmlwdCBzZXR1cD5cbjwvc2NyaXB0PlxuIiwiZWxlbWVudC1wbHVzLmpzIjoiaW1wb3J0IEVsZW1lbnRQbHVzIGZyb20gJ2VsZW1lbnQtcGx1cydcbmltcG9ydCB7IGdldEN1cnJlbnRJbnN0YW5jZSB9IGZyb20gJ3Z1ZSdcblxubGV0IGluc3RhbGxlZCA9IGZhbHNlXG5hd2FpdCBsb2FkU3R5bGUoKVxuXG5leHBvcnQgZnVuY3Rpb24gc2V0dXBFbGVtZW50UGx1cygpIHtcbiAgaWYgKGluc3RhbGxlZCkgcmV0dXJuXG4gIGNvbnN0IGluc3RhbmNlID0gZ2V0Q3VycmVudEluc3RhbmNlKClcbiAgaW5zdGFuY2UuYXBwQ29udGV4dC5hcHAudXNlKEVsZW1lbnRQbHVzKVxuICBpbnN0YWxsZWQgPSB0cnVlXG59XG5cbmV4cG9ydCBmdW5jdGlvbiBsb2FkU3R5bGUoKSB7XG4gIGNvbnN0IHN0eWxlcyA9IFsnaHR0cHM6Ly9wcmV2aWV3LTE4OTU4LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguY3NzJywgJ2h0dHBzOi8vcHJldmlldy0xODk1OC1lbGVtZW50LXBsdXMuc3VyZ2Uuc2gvYnVuZGxlL2luZGV4LmNzcyddLm1hcCgoc3R5bGUpID0+IHtcbiAgICByZXR1cm4gbmV3IFByb21pc2UoKHJlc29sdmUsIHJlamVjdCkgPT4ge1xuICAgICAgY29uc3QgbGluayA9IGRvY3VtZW50LmNyZWF0ZUVsZW1lbnQoJ2xpbmsnKVxuICAgICAgbGluay5yZWwgPSAnc3R5bGVzaGVldCdcbiAgICAgIGxpbmsuaHJlZiA9IHN0eWxlXG4gICAgICBsaW5rLmFkZEV2ZW50TGlzdGVuZXIoJ2xvYWQnLCByZXNvbHZlKVxuICAgICAgbGluay5hZGRFdmVudExpc3RlbmVyKCdlcnJvcicsIHJlamVjdClcbiAgICAgIGRvY3VtZW50LmJvZHkuYXBwZW5kKGxpbmspXG4gICAgfSlcbiAgfSlcbiAgcmV0dXJuIFByb21pc2UuYWxsU2V0dGxlZChzdHlsZXMpXG59XG4iLCJ0c2NvbmZpZy5qc29uIjoie1xuICBcImNvbXBpbGVyT3B0aW9uc1wiOiB7XG4gICAgXCJ0YXJnZXRcIjogXCJFU05leHRcIixcbiAgICBcImpzeFwiOiBcInByZXNlcnZlXCIsXG4gICAgXCJtb2R1bGVcIjogXCJFU05leHRcIixcbiAgICBcIm1vZHVsZVJlc29sdXRpb25cIjogXCJCdW5kbGVyXCIsXG4gICAgXCJ0eXBlc1wiOiBbXCJlbGVtZW50LXBsdXMvZ2xvYmFsLmQudHNcIl0sXG4gICAgXCJhbGxvd0ltcG9ydGluZ1RzRXh0ZW5zaW9uc1wiOiB0cnVlLFxuICAgIFwiYWxsb3dKc1wiOiB0cnVlLFxuICAgIFwiY2hlY2tKc1wiOiB0cnVlXG4gIH0sXG4gIFwidnVlQ29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcInRhcmdldFwiOiAzLjNcbiAgfVxufVxuIiwiUGxheWdyb3VuZE1haW4udnVlIjoiPHNjcmlwdCBzZXR1cD5cbmltcG9ydCBBcHAgZnJvbSAnLi9BcHAudnVlJ1xuaW1wb3J0IHsgc2V0dXBFbGVtZW50UGx1cyB9IGZyb20gJy4vZWxlbWVudC1wbHVzLmpzJ1xuc2V0dXBFbGVtZW50UGx1cygpXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8QXBwIC8+XG48L3RlbXBsYXRlPlxuIiwiaW1wb3J0LW1hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge1xuICAgIFwidnVlXCI6IFwiaHR0cHM6Ly9mYXN0bHkuanNkZWxpdnIubmV0L25wbS9AdnVlL3J1bnRpbWUtZG9tQGxhdGVzdC9kaXN0L3J1bnRpbWUtZG9tLmVzbS1icm93c2VyLmpzXCIsXG4gICAgXCJAdnVlL3NoYXJlZFwiOiBcImh0dHBzOi8vZmFzdGx5LmpzZGVsaXZyLm5ldC9ucG0vQHZ1ZS9zaGFyZWRAbGF0ZXN0L2Rpc3Qvc2hhcmVkLmVzbS1idW5kbGVyLmpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXNcIjogXCJodHRwczovL3ByZXZpZXctMTg5NTgtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5mdWxsLm1pbi5tanNcIixcbiAgICBcImVsZW1lbnQtcGx1cy9cIjogXCJ1bnN1cHBvcnRlZFwiLFxuICAgIFwiQGVsZW1lbnQtcGx1cy9pY29ucy12dWVcIjogXCJodHRwczovL2Zhc3RseS5qc2RlbGl2ci5uZXQvbnBtL0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlQDIvZGlzdC9pbmRleC5taW4uanNcIlxuICB9LFxuICBcInNjb3Blc1wiOiB7fVxufSIsIl9vIjp7InNob3dIaWRkZW4iOnRydWUsInN0eWxlU291cmNlIjoiaHR0cHM6Ly9wcmV2aWV3LTE4OTU4LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguY3NzIn19)

gyc-up commented:
> 似乎还是存在一些问题
> 
> [演示](https://element-plus.run/#eyJBcHAudnVlIjoiXG5cbjx0ZW1wbGF0ZT5cbiAgPGVsLWNhcmQgc3R5bGU9XCJtYXgtd2lkdGg6IDEwMCVcIj5cbiAgICA8ZWwtZGVzY3JpcHRpb25zIHRpdGxlPVwiQ3VzdG9taXplZCBzdHlsZSBsaXN0XCIgOmNvbHVtbj1cIjNcIiBib3JkZXI+XG4gICAgICA8ZWwtZGVzY3JpcHRpb25zLWl0ZW1cbiAgICAgICAgICBsYWJlbD1cInh4eHh4eFwiXG4gICAgICAgICAgbGFiZWwtYWxpZ249XCJyaWdodFwiXG4gICAgICAgICAgYWxpZ249XCJjZW50ZXJcIlxuICAgICAgICAgIGxhYmVsLWNsYXNzLW5hbWU9XCJteS1sYWJlbFwiXG4gICAgICAgICAgY2xhc3MtbmFtZT1cIm15LWNvbnRlbnRcIlxuICAgICAgICAgIHdpZHRoPVwiMTUwcHhcIlxuICAgICAgPlxuICAgICAgICBrb29yaW9va2FtaVxuICAgICAgPC9lbC1kZXNjcmlwdGlvbnMtaXRlbT5cbiAgICAgIDxlbC1kZXNjcmlwdGlvbnMtaXRlbSBsYWJlbD1cInh4eHhcIiBsYWJlbC1hbGlnbj1cInJpZ2h0XCIgYWxpZ249XCJjZW50ZXJcIj5cbiAgICAgICAgMTgxMDAwMDAwMDBcbiAgICAgIDwvZWwtZGVzY3JpcHRpb25zLWl0ZW0+XG4gICAgICA8ZWwtZGVzY3JpcHRpb25zLWl0ZW0gbGFiZWw9XCJ4eHhcIiBsYWJlbC1hbGlnbj1cInJpZ2h0XCIgYWxpZ249XCJjZW50ZXJcIj5cbiAgICAgICAgU3V6aG91XG4gICAgICA8L2VsLWRlc2NyaXB0aW9ucy1pdGVtPlxuICAgICAgPGVsLWRlc2NyaXB0aW9ucy1pdGVtIGxhYmVsPVwieHh4eFwiIGxhYmVsLWFsaWduPVwicmlnaHRcIiBhbGlnbj1cInJpZ2h0XCI+XG4gICAgICAgIDxlbC10YWcgc2l6ZT1cInNtYWxsXCI+U2Nob29sPC9lbC10YWc+XG4gICAgICA8L2VsLWRlc2NyaXB0aW9ucy1pdGVtPlxuICAgICAgPGVsLWRlc2NyaXB0aW9ucy1pdGVtIGxhYmVsPVwieHh4eHh4XCIgbGFiZWwtYWxpZ249XCJyaWdodFwiIGFsaWduPVwiY2VudGVyXCIgc3Bhbj1cIjJcIiByb3dzcGFuPVwiMlwiPlxuICAgICAgICBOby4xMTg4LCBXdXpob25nIEF2ZW51ZSwgV3V6aG9uZyBEaXN0cmljdCwgU3V6aG91LCBKaWFuZ3N1IFByb3ZpbmNlXG4gICAgICA8L2VsLWRlc2NyaXB0aW9ucy1pdGVtPlxuICAgICAgPGVsLWRlc2NyaXB0aW9ucy1pdGVtIGxhYmVsPVwieHh4eHh4eHhcIiBsYWJlbC1hbGlnbj1cInJpZ2h0XCIgYWxpZ249XCJjZW50ZXJcIj5cbiAgICAgICAgPGVsLXRhZyBzaXplPVwic21hbGxcIj5TY2hvb2w8L2VsLXRhZz5cbiAgICAgIDwvZWwtZGVzY3JpcHRpb25zLWl0ZW0+XG4gICAgICA8ZWwtZGVzY3JpcHRpb25zLWl0ZW0gbGFiZWw9XCJ4eHh4eHhcIiBsYWJlbC1hbGlnbj1cInJpZ2h0XCIgYWxpZ249XCJjZW50ZXJcIj5cbiAgICAgICAgTm8uMTE4OCwgV3V6aG9uZyBBdmVudWUsIFd1emhvbmcgRGlzdHJpY3QsIFN1emhvdSwgSmlhbmdzdSBQcm92aW5jZVxuICAgICAgPC9lbC1kZXNjcmlwdGlvbnMtaXRlbT5cbiAgICA8L2VsLWRlc2NyaXB0aW9ucz5cbiAgPC9lbC1jYXJkPlxuXG5cblxuPC90ZW1wbGF0ZT5cblxuPHN0eWxlIHNjb3BlZD5cbjpkZWVwKC5teS1sYWJlbCkge1xuICBiYWNrZ3JvdW5kOiB2YXIoLS1lbC1jb2xvci1zdWNjZXNzLWxpZ2h0LTkpICFpbXBvcnRhbnQ7XG59XG46ZGVlcCgubXktY29udGVudCkge1xuICBiYWNrZ3JvdW5kOiB2YXIoLS1lbC1jb2xvci1kYW5nZXItbGlnaHQtOSk7XG59XG48L3N0eWxlPlxuPHNjcmlwdCBzZXR1cD5cbjwvc2NyaXB0PlxuIiwiZWxlbWVudC1wbHVzLmpzIjoiaW1wb3J0IEVsZW1lbnRQbHVzIGZyb20gJ2VsZW1lbnQtcGx1cydcbmltcG9ydCB7IGdldEN1cnJlbnRJbnN0YW5jZSB9IGZyb20gJ3Z1ZSdcblxubGV0IGluc3RhbGxlZCA9IGZhbHNlXG5hd2FpdCBsb2FkU3R5bGUoKVxuXG5leHBvcnQgZnVuY3Rpb24gc2V0dXBFbGVtZW50UGx1cygpIHtcbiAgaWYgKGluc3RhbGxlZCkgcmV0dXJuXG4gIGNvbnN0IGluc3RhbmNlID0gZ2V0Q3VycmVudEluc3RhbmNlKClcbiAgaW5zdGFuY2UuYXBwQ29udGV4dC5hcHAudXNlKEVsZW1lbnRQbHVzKVxuICBpbnN0YWxsZWQgPSB0cnVlXG59XG5cbmV4cG9ydCBmdW5jdGlvbiBsb2FkU3R5bGUoKSB7XG4gIGNvbnN0IHN0eWxlcyA9IFsnaHR0cHM6Ly9wcmV2aWV3LTE4OTU4LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguY3NzJywgJ2h0dHBzOi8vcHJldmlldy0xODk1OC1lbGVtZW50LXBsdXMuc3VyZ2Uuc2gvYnVuZGxlL2luZGV4LmNzcyddLm1hcCgoc3R5bGUpID0+IHtcbiAgICByZXR1cm4gbmV3IFByb21pc2UoKHJlc29sdmUsIHJlamVjdCkgPT4ge1xuICAgICAgY29uc3QgbGluayA9IGRvY3VtZW50LmNyZWF0ZUVsZW1lbnQoJ2xpbmsnKVxuICAgICAgbGluay5yZWwgPSAnc3R5bGVzaGVldCdcbiAgICAgIGxpbmsuaHJlZiA9IHN0eWxlXG4gICAgICBsaW5rLmFkZEV2ZW50TGlzdGVuZXIoJ2xvYWQnLCByZXNvbHZlKVxuICAgICAgbGluay5hZGRFdmVudExpc3RlbmVyKCdlcnJvcicsIHJlamVjdClcbiAgICAgIGRvY3VtZW50LmJvZHkuYXBwZW5kKGxpbmspXG4gICAgfSlcbiAgfSlcbiAgcmV0dXJuIFByb21pc2UuYWxsU2V0dGxlZChzdHlsZXMpXG59XG4iLCJ0c2NvbmZpZy5qc29uIjoie1xuICBcImNvbXBpbGVyT3B0aW9uc1wiOiB7XG4gICAgXCJ0YXJnZXRcIjogXCJFU05leHRcIixcbiAgICBcImpzeFwiOiBcInByZXNlcnZlXCIsXG4gICAgXCJtb2R1bGVcIjogXCJFU05leHRcIixcbiAgICBcIm1vZHVsZVJlc29sdXRpb25cIjogXCJCdW5kbGVyXCIsXG4gICAgXCJ0eXBlc1wiOiBbXCJlbGVtZW50LXBsdXMvZ2xvYmFsLmQudHNcIl0sXG4gICAgXCJhbGxvd0ltcG9ydGluZ1RzRXh0ZW5zaW9uc1wiOiB0cnVlLFxuICAgIFwiYWxsb3dKc1wiOiB0cnVlLFxuICAgIFwiY2hlY2tKc1wiOiB0cnVlXG4gIH0sXG4gIFwidnVlQ29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcInRhcmdldFwiOiAzLjNcbiAgfVxufVxuIiwiUGxheWdyb3VuZE1haW4udnVlIjoiPHNjcmlwdCBzZXR1cD5cbmltcG9ydCBBcHAgZnJvbSAnLi9BcHAudnVlJ1xuaW1wb3J0IHsgc2V0dXBFbGVtZW50UGx1cyB9IGZyb20gJy4vZWxlbWVudC1wbHVzLmpzJ1xuc2V0dXBFbGVtZW50UGx1cygpXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8QXBwIC8+XG48L3RlbXBsYXRlPlxuIiwiaW1wb3J0LW1hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge1xuICAgIFwidnVlXCI6IFwiaHR0cHM6Ly9mYXN0bHkuanNkZWxpdnIubmV0L25wbS9AdnVlL3J1bnRpbWUtZG9tQGxhdGVzdC9kaXN0L3J1bnRpbWUtZG9tLmVzbS1icm93c2VyLmpzXCIsXG4gICAgXCJAdnVlL3NoYXJlZFwiOiBcImh0dHBzOi8vZmFzdGx5LmpzZGVsaXZyLm5ldC9ucG0vQHZ1ZS9zaGFyZWRAbGF0ZXN0L2Rpc3Qvc2hhcmVkLmVzbS1idW5kbGVyLmpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXNcIjogXCJodHRwczovL3ByZXZpZXctMTg5NTgtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5mdWxsLm1pbi5tanNcIixcbiAgICBcImVsZW1lbnQtcGx1cy9cIjogXCJ1bnN1cHBvcnRlZFwiLFxuICAgIFwiQGVsZW1lbnQtcGx1cy9pY29ucy12dWVcIjogXCJodHRwczovL2Zhc3RseS5qc2RlbGl2ci5uZXQvbnBtL0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlQDIvZGlzdC9pbmRleC5taW4uanNcIlxuICB9LFxuICBcInNjb3Blc1wiOiB7fVxufSIsIl9vIjp7InNob3dIaWRkZW4iOnRydWUsInN0eWxlU291cmNlIjoiaHR0cHM6Ly9wcmV2aWV3LTE4OTU4LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguY3NzIn19)

是最后一行应该换行处理吗，改动之前descriptions的实现效果就是这个样子，只是修改了一下跨行时，rowspan没有占据位置
还不清楚这个原本是不是想要实现的就是这个demo效果

tolking commented:
应该换行才符合预期吧。可能是之前就一直存在的问题

gyc-up committed:
fix(components): 针对于description中的span计算，在当前span小于column的情况下应该换行而没有换行

descriptio中的getRows函数计算span总数和当前的行数错误，导致应该换行的情况下没有换行

BREAKING CHANGE :
修改了description中的getRows函数，重写了计算逻辑

closed #18956

gyc-up commented:
> 应该换行才符合预期吧。可能是之前就一直存在的问题

这个换行改了一下，但是感觉如果按照这个预期的话，这个组件还有其他的一些bug  QAQ

wangwang93517 commented:
这段代码显示错乱了 

```
 <el-descriptions :column="4" border size="large" label-width="100px">
  <el-descriptions-item label="1-1"> </el-descriptions-item>
  <el-descriptions-item label="1-2"> </el-descriptions-item>
  <el-descriptions-item label="1-3"> </el-descriptions-item>
  <el-descriptions-item label="1-4"> </el-descriptions-item>
  <el-descriptions-item label="2&3-1" :span="2" :rowspan="2"> </el-descriptions-item>
  <el-descriptions-item label="2-2" :span="2" > </el-descriptions-item>
  <el-descriptions-item label="3-2" :span="2" > </el-descriptions-item>
  <el-descriptions-item label="4-1～4" :span="4"> </el-descriptions-item>
</el-descriptions>
```
